clients/lsp-yaml.el

```diff
@@ -171,18 +171,18 @@
   "Download the remote schema store at `lsp-yaml-schema-store-uri' into local cache.
 Set FORCE-DOWNLOADING to non-nil to force re-download the database."
   (interactive "P")
-  (or force-downloading
-      (file-exists-p lsp-yaml-schema-store-local-db)
-      (url-copy-file lsp-yaml-schema-store-uri lsp-yaml-schema-store-local-db)))
+  (when (or force-downloading (not (file-exists-p lsp-yaml-schema-store-local-db)))
+    (if (not (file-directory-p (file-name-directory lsp-yaml-schema-store-local-db)))
```

Comment:
I guess you can use `unless` instead of `if` and `not` here.

```suggestion
    (unless (file-directory-p (file-name-directory lsp-yaml-schema-store-local-db))
```

Reply:
Can you help us fix the indentation the line below? Thanks!

Reply:
Updated! Thanks.